### DISCUSSION TITLE
blog/2022/220428-how-to-ask-questions-like-a-retard/

### DISCUSSION THREAD

**Discussion by giscus** [zhanbao2000/blog#3] [Announcements]
# blog/2022/220428-how-to-ask-questions-like-a-retard/

这篇文章致力于告诉读者如何像一个弱智一样提问，以及如何在提问时激怒你的回答者。

https://blog.arisa.moe/blog/2022/220428-how-to-ask-questions-like-a-retard/

**Comment by MBRjun** [comment]
> *怎么打不开了*

**Comment by QR-Not-Code** [comment]
试过了，还是不行。有没有更加简单又强势的方法成为弱智？

**Reply by yu2256140203** [reply]
有的兄弟，有的，这样的方法当然不止一个

**Comment by yqs112358** [comment]
UP你这做的什么东西啊，我下下来打开想玩这个游戏怎么提示要付钱？这么黑？🤮快点把原版发给我，不然举报你了！！！

**Reply by zhanbao2000** [reply]
投降喵🙌

**Comment by sakimidare** [comment]
I DONT GIVE A FUCK ABOUT THE FUCKING CODE! i just want to download this stupid fucking application and use it

WHY IS THERE CODE??? MAKE A FUCKING .EXE FILE AND GIVE IT TO ME. these dumbfucks think that everyone is a developer and understands code. well i am not and i don't understand it. I only know to download and install applications. SO WHY THE FUCK IS THERE CODE? make an EXE file and give it to me. STUPID FUCKING SMELLY NERDS

**Comment by niko12344-ls** [comment]
这个方法太强势了 已经成为弱智了

**Comment by david082321** [comment]
TL;DR 所以要如何像一个弱智一样提问？[doge]🐶
